docs/src/getting-started.rst

```diff
@@ -1,876 +1,877 @@
+
+This command executes the |cdk| app and synthesize an |CFN| template for the
+**hello-cdk** stack:
+
+.. code-block:: yaml
+
+    Resources:
+        MyFirstBucketB8884501:
+            Type: 'AWS::S3::Bucket'
+            Properties:
+                VersioningConfiguration:
+                    Status: Enabled
+        CDKMetadata:
+            Type: 'AWS::CDK::Metadata'
+            Properties:
+                Modules: # ...
+
+You can see that the stack contains an **AWS::S3::Bucket** resource with the desired
+versioning configuration.
+
+.. note::
+
+    The **AWS::CDK::Metadata** resource was automatically added to your template
+    by the toolkit. This allows us to learn which libraries were used in your
+    stack. See :ref:`version-reporting` for more details and how to
+    :ref:`opt-out <version-reporting-opt-out>`.
+
+.. _deploy_stack:
+
+Deploying the Stack
+===================
+
+Use **cdk deploy** to deploy the stack:
+
+.. code-block:: sh
+
+    cdk deploy
+
+The **deploy** command synthesizes an |CFN| template from the stack
+and then invokes the |CFN| create/update API to deploy it into your AWS
+account. The command displays information as it progresses.
+
+.. _modify_cde:
+
+Modifying the Code
+==================
+
+Configure the bucket to use KMS managed encryption:
+
+.. tabs::
+
+    .. group-tab:: JavaScript
+
+        .. code-block:: js
+            :emphasize-lines: 3
+
+            new s3.Bucket(this, 'MyFirstBucket', {
+                versioned: true,
+                encryption: s3.BucketEncryption.KmsManaged
+            });
+
+    .. group-tab:: TypeScript
+
+        .. code-block:: ts
+            :emphasize-lines: 3
+
+            new s3.Bucket(this, 'MyFirstBucket', {
+                versioned: true,
+                encryption: s3.BucketEncryption.KmsManaged
+            });
+
+    .. group-tab:: Java
+
+        .. code-block:: java
+            :emphasize-lines: 3
+
+            new Bucket(this, "MyFirstBucket", BucketProps.builder()
+                    .withVersioned(true)
+                    .withEncryption("MANAGED")
+                    .build());
+
+Compile the program:
+
+.. tabs::
+
+    .. group-tab:: JavaScript
+
+        Nothing to compile.
+
+    .. group-tab:: TypeScript
+
+        .. code-block:: sh
+
+            npm run build
+
+    .. group-tab:: Java
+
+        .. code-block:: sh
+
+            mvn compile
+
+.. _prepare_deployment:
+
+Preparing for Deployment
+========================
+
+Before you deploy the updated stack, use the ``cdk diff`` command to evaluate
+the difference between the |cdk| app and the deployed stack:
+
+.. code-block:: sh
+
+    cdk diff
+
+The toolkit queries your AWS account for the current |CFN| template for the
+**hello-cdk** stack, and compares the result with the template synthesized from the app.
+The output should look like the following:
+
+.. code-block:: sh
+
+    [~] ?? Updating MyFirstBucketB8884501 (type: AWS::S3::Bucket)

```

Comment:
Your unicode broke here.

Reply:
Hmm, not sure where that came from. I'll fix